openedx/features/edly/cookies.py

```diff
@@ -0,0 +1,69 @@
+from django.conf import settings
+
+from openedx.core.djangoapps.user_authn.cookies import standard_cookie_settings
+from openedx.features.edly.models import EdlySubOrganization
+from openedx.features.edly.utils import encode_edly_user_info_cookie
+
+
+def set_logged_in_edly_cookie(request, response, user):
+    """
+    Set edly user info cookie at the time of user login.
```

Comment:
`Set cookies for edly users at the time of login.`

Reply:
done